src/utils/defaultSxProp.tsx

```diff
@@ -0,0 +1,5 @@
+import {BetterSystemStyleObject} from '../sx'
+
+export const defaultSxProp: Readonly<BetterSystemStyleObject> = __DEV__
+  ? Object.freeze<BetterSystemStyleObject>({})
+  : {}
```

Comment:
@joshblack froze this in dev, but typed it readonly as well, so it should help catch these easily